app/api/v2/identity/sessions.rb

```diff
@@ -28,6 +33,8 @@ class Sessions < Grape::API
           verify_captcha!(user: user,
                           response: params['recaptcha_response']) if declared_params[:recaptcha_response]
 
+          verify_geetest!(response: params['geetest_response']) if declared_params[:geetest_response]
```

Comment:
here should be helper method `verify_captcha`
In the helpers file (utils.rb) you can check the configuration.
> skip if disabled
>  if its recaptcha - call recaptcha_verifier
>  if its geetest -  check if all the params are present (challenge, validate, seccode) and call geetest verifier